core/src/main/java/hudson/model/details/DurationDetail.java

```diff
@@ -0,0 +1,15 @@
+package hudson.model.details;
+
+import hudson.model.Run;
+import jenkins.model.Detail;
+
+/**
+ * Displays the duration of the given run, or, if the run has completed, shows the total time it took to execute
+ * @implNote This will render Jelly, hence the fields return null
```

Comment:
not sure if this is needed,  no null fields anymore
```suggestion
```
maybe
```suggestion
 * @implNote This renders Jelly
```